cmd/finch/virtual_machine_settings.go

```diff
@@ -0,0 +1,83 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/runfinch/finch/pkg/config"
+	"github.com/runfinch/finch/pkg/flog"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newSettingsVMCommand(
+	logger flog.Logger,
+	lca config.LimaConfigApplier,
+	fs afero.Fs,
+	stdout io.Writer,
+) *cobra.Command {
+	settingsVMCommand := &cobra.Command{
+		Use:   "settings",
+		Short: "Configure the virtual machine instance",
+		RunE:  newSettingsVMAction(logger, lca, fs, stdout).runAdapter,
+	}
+
+	settingsVMCommand.Flags().Int("cpus", 0, "the amount of vCPU to dedicate to the virtual machine (restart the vm when applying this change.)")
```

Comment:
@pendo324  Should also support other configs? May be we can start with these 2 but i was thinking if we it is ok to support all configs.